jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/FormsSizingTests.java

```diff
@@ -9,6 +9,7 @@
 import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
 import static io.github.epam.states.States.shouldBeLoggedIn;
 import static org.hamcrest.CoreMatchers.is;
+import static org.testng.Assert.assertTrue;
 import static org.testng.AssertJUnit.assertEquals;
```

Comment:
we can see junit assertion here, it's an error, could you please fix it because we just found out it